apps/search/src/components/ImportProductsToAlgolia.tsx

```diff
@@ -45,39 +36,29 @@ export const ImportProductsToAlgolia = () => {
     }
   }, [searchProvider]);
 
-  useEffect(() => {
-    if (!searchProvider || isAlgoliaImporting || products.length <= currentProductIndex) {
-      return;
-    }
-    (async () => {
-      setIsAlgoliaImporting(true);
-      const productsBatchStartIndex = currentProductIndex;
-      const productsBatchEndIndex = Math.min(currentProductIndex + BATCH_SIZE, products.length);
-      const productsBatch = products.slice(productsBatchStartIndex, productsBatchEndIndex);
-
-      await searchProvider.updatedBatchProducts(productsBatch);
-
-      setIsAlgoliaImporting(false);
-      setCurrentProductIndex(productsBatchEndIndex);
-    })();
-  }, [searchProvider, currentProductIndex, isAlgoliaImporting, products]);
-
   return (
-    <Box __cursor={started ? "wait" : "auto"}>
+    <Box>
       {searchProvider && algoliaConfigured ? (
         <Box>
           <Text variant={"heading"} as={"p"} marginBottom={4}>
             Importing products & variants
           </Text>
           <Text as={"p"}>
-            Trigger initial indexing for products catalogue. It can take few minutes.{" "}
+            Trigger initial indexing for products catalogue. It can take few minutes and will run in
+            the background
           </Text>
-          <Text marginBottom={8} variant={"bodyStrong"}>
-            Do not close the app - its running client-side
-          </Text>
-          <Box display={"flex"} justifyContent={"flex-end"}>
-            <Button disabled={started || !searchProvider} onClick={importProducts}>
-              Start importing
+          <Box display={"flex"} justifyContent={"flex-end"} marginTop={13}>
+            <Button onClick={() => fetch("/api/index-products")}>Start importing</Button>
+            <Button
+              onClick={() =>
+                fetch("/api/jobs")
+                  .then((r: any) => r.json())
+                  .then((jobs: unknown) => {
+                    console.log(jobs);
+                  })
```

Comment:
not implemented yet. Its meant to pull job status, but jobs are not stored in the queue so we need to store them in our db